### PR TITLE
feat: open tabs not only by clicking on SfTab

### LINKED ISSUE #1742: [FEATURE] SfTabs - change tabs not only by clicking on SfTab
**Describe how this feature will help us improve**
Now SfTabs can be changed by clicking on a particular tab. There are some cases where clicking somewhere else (on a button) could change a tab

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes #1742

# Scope of work

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by AdamPawlinski** [comment]
@justyna-13 according tome solution can be to add emit event in toggle function in SfTabs and then listen to it in story like this:
SfTabs
```toggle(id) {
      this.$children.forEach((child) => {
        child._uid === id ? (child.isActive = true) : (child.isActive = false);        
      });
      const active = this.$children.findIndex(
        child => child.isActive === true
      ) + 1;
      this.$emit('click:tab', active)
    },
```
story: 
```
<SfTabs
    :open-tab="open"
    :tabMaxContentHeight="tabMaxContentHeight"
    :tabShowText="tabShowText"
    :tabHideText="tabHideText"
    @click:tab="change"
  >
    <SfTab
        v-for="tab in tabs"
        :key="tab.title"
        :title="tab.title"
    >
        {{tab.content}}
    </SfTab>
  </SfTabs>
```

Let me know what you think?

File: packages/vue/src/components/organisms/SfTabs/SfTabs.vue [hunk 1]
```diff
@@ -38,17 +38,24 @@ export default {
...
     this.$on("toggle", this.toggle);
     if (this.openTab) this.openChild();
   },
   methods: {
     toggle(id) {
       this.$children.forEach((child) => {
-        child._uid === id
-          ? (child.isActive = !child.isActive)
-          : (child.isActive = false);
+        child._uid === id ? (child.isActive = true) : (child.isActive = false);
```
Comment (Baroshem) [inline packages/vue/src/components/organisms/SfTabs/SfTabs.vue:54]:
```suggestion
        child.isActive = child._uid === id;
```

File: packages/vue/src/components/organisms/SfTabs/SfTabs.vue [hunk 2]
```diff
@@ -38,17 +38,24 @@ export default {
...
       });
+      const active =
```
Comment (Baroshem) [inline packages/vue/src/components/organisms/SfTabs/SfTabs.vue:56]: Maybe change name of the variable to something connected to index? I.e
```suggestion
      const activeTabId =
```

**Review by Baroshem — CHANGES_REQUESTED** [review]
Well done, it works now!

Some small suggestions from my side ;)

**Review by AdamPawlinski — APPROVED** [review]
I have no questions here. Maybe we can add this new behaviour to the product example. After clicking on "read all reviews" button the "read reviews" button could be opened. But we can do it in separate issue. Wdyt @justyna-13?

**Review by Baroshem — APPROVED** [review]

**Pull request merged**